distribution/helm/vector-agent/values.yaml

```diff
@@ -32,6 +32,33 @@ updateStrategy: RollingUpdate
 # Only used when updateStrategy is set to "RollingUpdate".
 maxUnavailable: 1
 
+# Use Liveness Probe
```

Comment:
One thing we may want to add as a comment is that this isn't available by default, and the user needs to enable the `api` themselves to get this endpoint exposed.